en/docs/install-and-setup/install-on-vm.md

```diff
@@ -7,55 +7,56 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
 
 ### Microgateway Toolkit
 
-1.  Log in to the command line (Terminal on Mac & Linux).
+1.  Download the [WSO2 API Microgateway toolkit distribution](https://wso2.com/api-management/api-microgateway/).
 
-2.  Download the [WSO2 API Microgateway toolkit distribution](https://wso2.com/api-management/api-microgateway/) .
-
-3.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Toolkit** , which will hereafter be referred to as `           <MGW-TK_HOME>          ` .
-
-4.  Append the full path of the `           /bin          ` folder of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
+2.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway Toolkit, which will hereafter be referred to as `<MGW-TK_HOME>`.
+    
+3.  Append the full path of the `/bin` folder of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
     This allows you to run the WSO2 API Microgateway toolkit related commands from any directory location.
-
-    -   [**Linux & Mac**](#toolkit-path-unix)
-    -   [**Windows**](#toolkit-path-windows)
-
-    **Example**
-
-    ``` java
+    
+    #### Setup toolkit path in Linux/MacOS
+    
+    ``` java tab="Format"
         export PATH=$PATH:<extracted_toolkit_location>/bin
     ```
-
-        Use one of the following approaches to define the paths.
-
-        -   Set the Microgateway toolkit distribution path using the following command in the terminal.
-
-        **Example**
-
-        ``` java
-            setx "%path%;<extracted_toolkit_location>/bin"
+    
+    #### Setup toolkit path in Windows
+
+    Use one of the following approaches to define the paths.
+    
+    1.  Set the Microgateway toolkit distribution path using the following command in the terminal.
+    
+        ``` java tab="Format"
+        setx "%path%;<extracted_toolkit_location>/bin"
         ```
-
-            -   Set the Microgateway toolkit distribution path using the UI in Windows.
-
-            !!! note
+    
+    2.  Set the Microgateway toolkit distribution path using the UI in Windows.
```

Comment:
I'm wondering if we need to provide this level of details instructions to add an item to `PATH` variable

Reply:
fixed https://github.com/wso2/docs-mg/pull/54/commits/22e1005ff35ccb1e2f1679aeeaba27e6d514bf6c